src/utils/custom-highlight-lang/swift.js

```diff
@@ -41,5 +41,37 @@ export default function swiftOverride(hljs) {
     };
   }
 
+  // Checks if a given language sub-mode matches the "ESCAPED_NEWLINE" from the
+  // built-in Swift parser from hljs
+  const isEscapedNewlineMode = (mode) => {
+    const { className, match } = mode;
+    if (className !== 'subst' || !match) {
+      return false;
+    }
+
+    const matchStr = match.toString();
+    return matchStr.startsWith('\\') && matchStr.endsWith('[\\t ]*(?:[\\r\\n]|\\r\\n)');
+  };
+
+  // replace the "ESCAPED_NEWLINE" sub-mode in the multiline string literal mode
+  // variants so that it doesn't include the actual newline characters in the
+  // span token that it generates, because this causes issues with our
+  // line-number + multi-line string literal logic when the span for the
+  // backslash token is split across multiple lines
+  const str = language.contains.find(({ className }) => className === 'string');
+  str.variants = str.variants.map(variant => ({
+    ...variant,
+    contains: variant.contains.map(mode => (isEscapedNewlineMode(mode) ? ({
+      className: 'subst',
+      begin: /\\/,
+      end: /[\t ]*(?:[\r\n]|\r\n)/,
```

Comment:
I guess this is the actual fix, right? Giving it a `start` and an `end`, instead of an open `match`?

Reply:
Also quickly added support for the optional extended `#` delimiters with 685c910e89d5671f8cbfa07edefdc96518b4f748